apps/recovery-portal/src/main/webapp/self-registration-complete.jsp

```diff
@@ -37,6 +51,42 @@
 
     isEmailNotificationEnabled = Boolean.parseBoolean(application.getInitParameter(
             IdentityManagementEndpointConstants.ConfigConstants.ENABLE_EMAIL_NOTIFICATION));
+
+    if(tenantDomain != null){
+        	 isAutoLoginEnable = Boolean.parseBoolean(Utils.getConnectorConfig("SelfRegistration.AutoLogin.Enable",
```

Comment:
Fix indentation in all the places

Reply:
Fixed [ed5eadb](https://github.com/wso2/identity-apps/pull/1409/commits/ed5eadbb4f3cd1e376311a9ace165eef81e761c6)